build-configs/obdach/index.ts

```diff
@@ -0,0 +1,109 @@
+import { OBDACH_ASSETS } from '../AssetsType'
+import {
+  AndroidBuildConfigType,
+  CommonBuildConfigType,
+  iOSBuildConfigType,
+  WebBuildConfigType
+} from '../BuildConfigType'
+import { APP_STORE_TEAM_ID } from '../common/constants'
+import mainImprint from './mainImprint'
+import { lightTheme } from './theme'
+
+const APPLE_APP_ID = '1072353915'
+
+const commonObdachBuildConfig: CommonBuildConfigType = {
+  appName: 'Vernetztes Obdach',
+  appIcon: 'app_icon_obdach',
+  lightTheme,
+  assets: OBDACH_ASSETS,
+  cmsUrl: 'https://admin.vernetztesobdach.de',
```

Comment:
This is quite sure not the cms api url but the cms frontend. Should probably be `https://cms.vernetztesobdach.de`